test/integration_helper.cr

```diff
@@ -44,9 +45,22 @@ class Minitest::Test
     create_git_repository "release", "0.2.0", "0.2.1", "0.2.2"
     create_git_release "release", "0.3.0", "name: release\nversion: 0.3.0\ncustom_dependencies:\n  pg:\n    git: #{git_path("optional")}\n"
 
-    create_git_repository "empty"
-    create_git_commit "empty", "initial release"
+    # git dependencies with prereleases:
+    create_git_repository "unstable", "0.1.0", "0.2.0", "0.3.0.alpha", "0.3.0.beta"
+    create_git_repository "preview", "0.1.0", "0.2.0", "0.3.0.a", "0.3.0.b", "0.3.0", "0.4.0.a"
+
+    # path dependency:
+    create_path_repository "foo", "0.1.0"
+
+    # dependency with neither a shard.yml and/or version tags:
+    #create_git_repository "empty"
```

Comment:
Why is this disabled?

Reply:
Not used anymore, but could be brought back (to test errors for example).